tools/run_tests/run_tests.py

```diff
@@ -712,7 +713,7 @@ def test_specs(self):
         runtime_cmd = ['mono']
 
     specs = []
-    for assembly in tests_by_assembly.iterkeys():
```

Comment:
Just drop the `.iterkeys()`entirely and go with `for assembly in tests_by_assembly:`?